src/test/tests.scala

```diff
@@ -109,5 +111,62 @@ object Main extends Suite("Probably Tests") {
       }.assert(_ == 2)
       runner.report
     }.assert(_.results.head.count == 100)
+
+    test("assert without predicate should succeed") {
+      val runner = new Runner()
+      runner("test division") {
+        5 / 2
+      }.assert()
+      runner.report()
+    }.assert(r => r.passed == r.total)
```

Comment:
Probably best to check that the total and the number of passed tests are both one.
```suggestion
    }.assert(_.passed == 1, _.total == 1)
```